monopod/pkg/commands/scaffold.go

```diff
@@ -91,192 +67,70 @@ func Scaffold() *cobra.Command {
 
 // runScaffold does the real work of creating the folders, files, and evaluating
 // templates.
-func (o scaffoldOptions) runScaffold() error {
+func (o scaffoldOptions) runScaffold(ctx context.Context) error {
```

Comment:
This context doesn't look like it's in use right now. Is it a good practice to always have it, even when not being leveraged?

Reply:
You're right. I'd piped it through out of habit, and then forgot to use it for anything. 

I'd say it's nice to have it already piped-through, even if you're not using it, just so there's one less excuse to use `context.Background()` if you need a context later.

I don't consider it a hard fast rule though.

Reply:
Yeah, having it wired through is nice. I'd change the `ctx` to `_` though?

Reply:
Agree with having it as `_`.